prow/cluster/jobs/IBM/multicloud-operators-placementrule/IBM.multicloud-operators-placementrule.master.yaml

```diff
@@ -0,0 +1,115 @@
+presubmits:
+  IBM/multicloud-operators-placementrule:
+  - name: build_multicloud-operators-placementrule
+    branches:
+    - ^master$
+    decorate: true
+    always_run: true
+    path_alias: github.com/IBM/multicloud-operators-placementrule
+    rerun_command: /test build
+    spec:
+      containers:
+      - command:
+        - make
+        - build
+        image: quay.io/IBM/multicloud-operators-placementrule:2019-09-25T13-11-16
```

Comment:
Can we reuse the current build-tools image?

Reply:
@kuanf